lib/net/ldap/connection.rb

```diff
@@ -3,6 +3,9 @@
 class Net::LDAP::Connection #:nodoc:
   include Net::LDAP::Instrumentation
 
+  # Seconds before failing for socket connect timeout
+  DefaultConnectTimeout = 1
```

Comment:
Sure, it's 5 seconds now.
I doubt more than that would be of any use, but let me know if you want to bump it further.